client.js

```diff
@@ -39,4 +43,25 @@ const employees = [
 // This is not a race. Everyone on your team should understand what is happening.
 // Ask questions when you don't.
 
-console.log( employees );
+
+
+
+// Loop over the employees array
+/*
+for(let i; i < ??; i++) {
+  // Call the function below, passing a single employee object as the parameter
+  // You will get back (see the return in the function) a new object. Log that new object to the console.
+}
+*/
+
+
+
+// this function will calculate 1 employee's bonus!
+function calculateIndividualEmployeeBonus( employee ) {
+  
+
+  
+  
+  // return new object with bonus results
+
+}
```

Comment:
Maybe this is too much guidance. How'd you feel about adding something to line 69 like:

```js
// It's often a good idea to "make it work for one thing" before
// making it work for all the things. In this case, we
// should get this function working for just one employee
// before we try calling it multiple times inside a loop.
// Let's start by getting this function to work on Atticus:
let bonusResults = calculateIndividualEmployeeBonus(employees[0);

console.log(bonusResults);
```

Reply:
I like it. are you suggesting just this or what would you remove?

Reply:
moved to readme